graphql/admin/admin.go

```diff
@@ -885,15 +892,18 @@ func (as *adminServer) lazyLoadSchema(namespace uint64) {
 		return
 	}
 
+	var generatedSchema schema.Schema
 	if sch.Schema == "" {
+		// if there was no schema stored in Dgraph, we still need to attach resolvers to the main
+		// graphql server which should just return errors for any incoming request.
+		// generatedSchema will be nil in this case
 		glog.Infof("No GraphQL schema in Dgraph; serving empty GraphQL API")
```

Comment:
Why has the return from here being removed ?

Reply:
If we return from here, then there will be no resolver attached to the main graphql server for the given namespace. Which is what results into panic when a query is sent for that namespace.